index.js

```diff
@@ -63,6 +66,12 @@ module.exports = (bookshelf, settings) => {
     initialize: function () {
       modelPrototype.initialize.call(this)
 
+      if (settings.sentinel) {
```

Comment:
I'm ok with that being added later - could even do something where you can use a boolean to make it global or a dictionary with table names whitelisted or blacklisted. Agree with your point on consistency of unique indexes.